com.dz4k.FruitCredits.json

```diff
@@ -0,0 +1,3397 @@
+        "runhaskell Setup build",
+        "runhaskell Setup install"
+      ],
+      "sources": [
+        {
+          "sha256": "0a8ec081956d18f8ca84f58aacf39a5ab1a7dc08302e2ce28b4ac3768bb0372c",
+          "type": "archive",
+          "url": "https://hackage.haskell.org/package/utility-ht-0.0.17.2/utility-ht-0.0.17.2.tar.gz"
+        },
+        {
+          "dest-filename": "utility-ht.cabal",
+          "sha256": "ee925ec38eebd2d6f0ff2017d3f9fecde6e27c07f29c64b5b4f9a678ff5fd15e",
+          "type": "file",
+          "url": "https://hackage.haskell.org/package/utility-ht-0.0.17.2/revision/0.cabal"
+        }
+      ]
+    },
+    {
+      "name": "wizards",
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "echo '#! /usr/bin/env runhaskell' >Setup.txt",
+        "echo '> import Distribution.Simple' >>Setup.txt",
+        "echo '> main = defaultMain' >>Setup.txt",
+        "ln -s Setup.txt Setup.lhs || true",
+        "runhaskell Setup configure --prefix=/app",
+        "runhaskell Setup build",
+        "runhaskell Setup install"
+      ],
+      "sources": [
+        {
+          "sha256": "05650d7bf6dd0e6b87d0d7da6fb003601ce5d7b1f3d69571127ec3c9425b9bb2",
+          "type": "archive",
+          "url": "https://hackage.haskell.org/package/wizards-1.0.3/wizards-1.0.3.tar.gz"
+        },
+        {
+          "dest-filename": "wizards.cabal",
+          "sha256": "cb7798895ee35e06c00a69eb1cd8a3c45921e3c5fb76ffcf11a76a15fbda4365",
+          "type": "file",
+          "url": "https://hackage.haskell.org/package/wizards-1.0.3/revision/3.cabal"
+        }
+      ]
+    },
+    {
+      "name": "hledger",
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "echo '#! /usr/bin/env runhaskell' >Setup.txt",
+        "echo '> import Distribution.Simple' >>Setup.txt",
+        "echo '> main = defaultMain' >>Setup.txt",
+        "ln -s Setup.txt Setup.lhs || true",
+        "runhaskell Setup configure --prefix=/app",
+        "runhaskell Setup build",
+        "runhaskell Setup install"
+      ],
+      "sources": [
+        {
+          "sha256": "679d56d1423d0986a4fe01fc802f44b74054a599f6bc5f66ec33bddad25d34a1",
+          "type": "archive",
+          "url": "https://hackage.haskell.org/package/hledger-1.40/hledger-1.40.tar.gz"
+        },
+        {
+          "dest-filename": "hledger.cabal",
+          "sha256": "ca66d0dbc8316c2613f54f9fb1b2e624eae8cf848807e4a085e0b276afb014ce",
+          "type": "file",
+          "url": "https://hackage.haskell.org/package/hledger-1.40/revision/0.cabal"
+        }
+      ]
+    },
+    {
+      "name": "fruit-credits",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "dir",
+          "path": "."
+        },
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/BZHDeveloper/vul.git",
+          "branch": "master",
```

Comment:
Never a git branch. You need a specific commit.

Reply:
Thanks and fixed.